src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,514 @@
+| TracerName                | REQUIRED. Name of the tracer compound used (for example, "CIMBI-36")                                                                                                                                                                                                                                                                                                                                                                          |
+| TracerRadionuclide        | REQUIRED. Radioisotope labelling tracer (for example, "C11").                                                                                                                                                                                                                                                                                                                                                                                 |
+| InstitutionName           | RECOMMENDED. The name of the institution in charge of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 0080 InstitutionName.                                                                                                                                                                                                                                                                               |
+| InstitutionAddress        | RECOMMENDED. The address of the institution in charge of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 0081 InstitutionAddress.                                                                                                                                                                                                                                                                         |
+| InstitutionDepartmentName | RECOMMENDED. The department in the institution in charge of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 1040 Institutional Department Name.                                                                                                                                                                                                                                                           |
+| BodyPart                  | RECOMMENDED. Name of the organ / body region scanned.                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Should we have a list of body parts? Is there a standardized list somewhere? This seems like something we will also need at some other points, maybe we can predefine the options

Reply:
Very good point! @effigies @sappelhoff @Remi-Gau @tsalo another thing to discuss in our meeting tomorrow.